includes/abstracts/abstract-wc-order.php

```diff
@@ -145,6 +145,41 @@ public function remove_order_items( $type = null ) {
 	}
 
 	/**
+	 * Returns a list of all payment tokens associated with the current order
+	 *
+	 * @since 2.6
+	 * @return array An array of payment token objects
+	 */
+	public function get_payment_tokens() {
+		return WC_Payment_Tokens::get_order_tokens( $this->id );
+	}
+
+	/**
+	 * Add a payment token to an order
+	 *
+	 * @since 2.6
+	 * @param  int                $token_id  ID for the token we are adding to the order
+	 * @param  WC_Payment_Token   $token     Payment token object
+	 * @return boolean True if the token was added, false if not
+	 */
+	public function add_payment_token( $token_id, $token ) {
+		if ( empty( $token ) || ! ( $token instanceof WC_Payment_Token ) || $token_id < 1 ) {
+			return false;
+		}
+
+		$wc_tokens_meta = get_post_meta( $this->id, '_wc_payment_tokens', true );
```

Comment:
Changing this variable name to be `$token_ids` would provide a more accurate description of this variable's contents as it contains an array of token IDs, not token meta data (it also doesn't really need the `wc_` prefix, because it's already inside of a WC class).